client/app/telecom/pack/slots/voipEcoFax/pack-voipEcoFax.controller.js

```diff
@@ -1,35 +1,56 @@
-angular.module('managerApp').controller('PackVoipEcoFaxCtrl', function ($scope, OvhApiPackXdslVoipEcofax, $stateParams, REDIRECT_URLS) {
-  const self = this;
+import _ from 'lodash';
 
-  const init = function init() {
-    self.details = $scope.service;
-    self.services = [];
+export default class PackVoipEcoFaxCtrl {
```

Comment:
Related to your commit 1ab9bc9, it is not necessary to export your class since you only use it in place.

Reply:
I've not turned it into a `component` because it is used with a `ng-include` and it would require a lot more refactoring. I still `export` the class in order to ease (slightly) migration even though for now it is not useful

Reply:
Nevermind, but just bear in mind that the migration was already almost done from that part; https://github.com/ovh-ux/manager/pull/152

/cc @cbourgois